pom.xml

```diff
@@ -87,6 +87,7 @@
 
 	<modules>
 		<module>spring-cloud-kubernetes-dependencies</module>
+		<module>spring-cloud-kubernetes-commons</module>
 		<module>spring-cloud-kubernetes-core</module>
```

Comment:
I don't think we need to deal with it right this second, but commons and core seem similar. Is core now fabric8 specific?